src/main/java/com/cloudogu/smeagol/wiki/infrastructure/GitClient.java

```diff
@@ -123,6 +126,21 @@ public File file(String path) {
         return Optional.empty();
     }
 
+    public List<RevCommit> findCommits(String path) throws IOException, GitAPIException {
+        Git git = open();
+        List<RevCommit> commits = new ArrayList<>();
+        Iterator<RevCommit> iterator = git.log()
+                .addPath(path)
+                .call()
+                .iterator();
+
+        if (iterator.hasNext()) {
+            iterator.forEachRemaining(revCommit -> commits.add(revCommit));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ecosystem.cloudogu.com/sonar/coding_rules#rule_key=squid%3AS1612)